### PR TITLE
Babylon Council #2 - Tokenomics Report

### PR DESCRIPTION
**Pull request by freakstatic**
Report generated by the Council Report Generator with the block range 1670401 - 1872001

### CODE REVIEW

**Comment by freakstatic** [comment]
Proposal link:
https://testnet.joystream.org/#/proposals/121

**Review by bwhm — APPROVED** [review]
Looks good!
Waiting for the Council to approve before merging though...

**Pull request merged**
